microsoft-edge/webview2/samples/webview2samplewincomp.md

```diff
@@ -73,159 +73,113 @@ Microsoft Visual Studio is required.  Microsoft Visual Studio Code is not suppor
-
-   <!-- Solution Explorer shows the **WebView2SampleWinComp** project: -->
-
-   <!-- ![The WebView2SampleWinComp sample opened in Visual Studio in Solution Explorer.](media/webview2samplewincomp-in-solution-explorer.png) -->
-   <!--todo: create png-->
+* In **Solution Explorer**, right-click the **WebView2SampleWinComp** project (not solution), and then click **Retarget Projects**.
 
 
 <!-- ====================================================================== -->
-## Step 7 - View the opened project
+## Step 6 - Install the Windows SDK
 
-The project opens in Visual Studio:
+By default, this sample app uses the latest Window 10 SDK version you have installed.  There's an issue with Windows 10 SDK, version 2004 (10.0.**19041**.0) that will stop this sample app from building.  If you run into this issue, either install (and retarget this project to use) a later version, such as Windows 10 SDK version 2104 (10.0.**20348**.0), or an earlier version, such as 10.0.**18362**.1.  
 
-![The WebView2SampleWinComp project in Visual Studio.](media/webview2samplewincomp-project-in-sln-explorer.png)
+To install a Windows 10 SDK:
 
-_To zoom, right-click > **Open image in new tab**._
+1. Go to [Windows SDK and emulator archive](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive/).
 
+1. On one of the following rows, click the **Install SDK** link:
+   *  Windows 10 SDK version 2104 (10.0.20348.0) (or later)
+   *  Windows 10 SDK, version 1903 (10.0.18362.1)
 
-<!-- ====================================================================== -->
-## Step 8 - Install or update the WebView2 SDK
+   A version-specific copy of `winsdksetup.exe` is downloaded to your `Downloads` directory.
 
-<!-- comment on sample says: "Update apps to to 1.0.1056-prerelease (#110)"  Oct 28 2021 -->
+1. In your `Downloads` directory, open the copy of `winsdksetup.exe` that you just downloaded.
 
-1. Install or update the WebView2 SDK on the project node (not the solution node) in Solution Explorer.  In a separate window or tab, see [Install the WebView2 SDK](../how-to/machine-setup.md#install-the-webview2-sdk) in _Set up your Dev environment for WebView2_.  Follow the steps in that section, and then return to this page and continue below.
+1.   The **Windows SDK setup** window opens:
 
+  ![Windows SDK setup.](webview2samplewincomp-images/windows-sdk-setup.png)
 
-<!-- ====================================================================== -->
-## Step 9 - Build the project
+1. Click the **Next** button and then follow the prompts.  You can accept the defaults.  At the end of installing, the Windows SDK Welcome screen for whichever version that you selected appears:
 
-At the top of Visual Studio, set the build target, as follows:
+  ![Welcome to the Windows SDK.](webview2samplewincomp-images/welcome-winsdk.png)
 
-1. In the **Solution Configurations** dropdown list, select **Debug** or **Release**.
+1. Click the **Close** button.
```

Comment:
Numbered steps in "Step 6" are not consecutive because images need to be indented.